### PR TITLE
Support writing mode

### PR DESCRIPTION
**Pull request by kevers-google**
With this change, all tests now pass in:

scroll-animations/current-time-writing-modes.html

### CODE REVIEW

File: src/scroll-timeline-base.js [hunk 1]
```diff
@@ -89,9 +115,14 @@ export function installScrollOffsetExtension(parseFunction, evaluateFunction) {
...
-  if (orientation === "block") orientation = "vertical";
-  else if (orientation === "inline") orientation = "horizontal";
+  // Only one horizontal writing mode: horizontal-tb.  All other writing modes
+  // flow vertically.
+  const horizontalWritingMode =
+    getComputedStyle(this.scrollSource).writingMode == 'horizontal-tb';
+  if (orientation === "block")
+    orientation = horizontalWritingMode ? "vertical" : "horizontal";
+  else if (orientation === "inline")
+    orientation = verticalWritingModel ? "vertical" : "horizontal";
```
Comment (flackr) [inline src/scroll-timeline-base.js:125]: verticalWritingModel is undefined. This probably points to a lack of tests covering this.

File: src/scroll-timeline-base.js [hunk 2]
```diff
@@ -58,6 +58,32 @@ function calculateTimeRange(scrollTimeline) {
...
+ * @param scrollTimeline {ScrollTimeline}
+ * @param orientation {String}
+ * @returns {Number}
+ */
+function directionAwareScrollOffset(scrollSource, orientation) {
+  const style = getComputedStyle(scrollSource);
+  // All writing modes are vertical except for horizontal-tb.
+  // http://drafts.csswg.org/css-writing-modes-4/#block-flow
+  const horizontalWritingMode = style.writingMode == 'horizontal-tb';
+  let currentScrollOffset  = scrollSource.scrollTop;
```
Comment (flackr) [inline src/scroll-timeline-base.js:73]: Looks like sieways-lr switches the inline direction to be from bottom to top, which may require flipping or taking the absolute value of the scrollTop. See https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode#Examples. We can add a TODO for this - it also seems to be unsupported in chrome.

**Review by flackr — CHANGES_REQUESTED** [review]

File: src/scroll-timeline-base.js (hunk 2 above)
Comment (kevers-google) [reply src/scroll-timeline-base.js:75]: Added a TODO.

File: src/scroll-timeline-base.js (hunk 1 above)
Comment (kevers-google) [reply src/scroll-timeline-base.js]: Oops.  Fixed.  Yes we have a hole in the test coverage.

**Review by flackr — APPROVED** [review]

**Pull request merged**